### PR TITLE
Updated install.sh, added elementary os to install

### PR DESCRIPTION
**Pull request by mozinrat**
Signed-off-by: Rohit Verma rohitatiit@gmail.com

Tested installation on elementary os freya beta, based on ubuntu 14.04.

### CODE REVIEW

**Comment by icecrime** [comment]
Ping @tianon!

File: hack/install.sh [hunk 1]
```diff
@@ -79,7 +79,7 @@ if [ -z "$lsb_dist" ] && [ -r /etc/os-release ]; then
 	lsb_dist="$(. /etc/os-release && echo "$ID")"
 fi
 
-lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
+lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]'|  cut -d' ' -f1)"
```
Comment (tianon) [inline hack/install.sh]: Why the extra `cut` here?

Comment (mozinrat) [inline hack/install.sh]: the distort is usually echoed as "elementary OS", the cut is to grep
elementary only.

**Comment by martyglaubitz** [comment]
any updates on this? i'm waiting on this to be merged

File: hack/install.sh (hunk 1 above)
Comment (tianon) [inline hack/install.sh]: So why not just put `'elementary os'` in the `case` below, especially so we don't accidentally break some other distribution later?
ie:
```bash
case "$lsb_dist" in
  # ...
  ubuntu|debian|linuxmint|'elementary os')
      # ...
```

**Comment by onbjerg** [comment]
I am also waiting for this to be merged.

**Comment by mozinrat** [comment]
@tianon in fact thats the first thing which I tried and that also works flawlessly. I was thinking about calling the os with their first name as well as try to eliminate quotes in case condition, since we don't have any so far. I leave the decision upto you, I don't see any flaw in either.

**Comment by tianon** [comment]
I'd rather we go with the quoted string in the case statement, especially since it's more explicit and no less correct.  This will help ensure we have less chance of potential breakage on some future OS with spaces in its name.

File: hack/install.sh (hunk 1 above)
Comment (axltxl) [inline hack/install.sh]: I totally support @tianon on this, any other approach would  be likely overkill ...

**Comment by jfrazelle** [comment]
closed by #12416

**Pull request closed without merging**